Address Denis' comments (#481)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

:  An entity that creates SD-JWTs.

Holder:

> :  An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term usually refers to the actual user and the supporting hardware and software in their possession.

Reviewer: I am not totally sure this change is needed or an improvement... ?

Reviewer 2: agree with Brian..

Reviewer 2: ```suggestion
: An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term may refer to the actual user, the supporting hardware and software in their possession, or both.
```

* a hash of the SD-JWT,
* a nonce to ensure the freshness of the signature, and

> * an audience value to indicate the intended Verifier or Verifiers for the document.

Reviewer: while "Verifier or Verifiers" is true I guess, I don't think we need to mention the plural here. 
```suggestion
* an audience value to indicate the intended Verifier for the document.
```

Note that these processing steps do not yield any guarantees to the Holder about having received a complete set of Disclosures. That is, for some digest values in the Issuer-signed JWT (which are not decoy digests) there may be no corresponding Disclosures, for example, if the message from the Issuer was truncated.

> 

Reviewer: ```suggestion
It is up to the Holder how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the user when needed.
```

Reviewer: or perhaps
```suggestion
It is up to the Holder's software how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the user when needed.
```

Author: Everything here is about the user, so I would go for the first version. I initially removed the sentence because it didn't add much value and deepens the potential Holder/user confusion. But I'm fine with keeping it or a suitable replacement.

Reviewer: I kinda presumed as much. I do think it adds some value so would like something suitable kept or replaced.

Reviewer 2: +1 to the first proposal of the addition

> It is up to the Holder how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the End-User when needed.

Reviewer: I sorta feel like something like this sentence should remain.

Reviewer: See https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/481/files#r1817324384 'cause github won't let me make a suggestion on deleted lines.
